docs/cli.md

```diff
@@ -1010,6 +1010,90 @@ command.  For example, to use the serveo.net, run the below command:
 fedify tunnel --service serveo.net 3000
 ~~~~
 
+`fedify webfinger`: Looking up a WebFinger resource
+-----------------------------------------------------
```

Comment:
It's trivial, but still …

```suggestion
---------------------------------------------------
```

Reply:
Changed at 99dbd04ae12d00763c20649eedcdb45e54ad3a7c!